File: Iniciante/1146.py
sqc = ""
primeiroLoop = True

while (True):
  x = int(input())

  if (x == 0):
    break

  if not primeiroLoop:
    sqc += "\n"

  for i in range(1, x+1):
    sqc += str(i)

    if (i < x):
      sqc += " "

  primeiroLoop = False

print(sqc)

File: Iniciante/1146.js
let sqc = "";
let primeiroLoop = true;

while (true) {
  let x = Number(lines.shift());

  if (x === 0) {
    break;
  }

  if (!primeiroLoop) {
    sqc += "\n";
  }

  for (let i = 1; i <= x; i++) {
    sqc += i;

    if (i < x) {
      sqc += " ";
    }
  }

  primeiroLoop = false;
}

console.log(sqc);
